.gitignore

```diff
@@ -1,38 +1,5 @@
-HELP.md
```

Comment:
Why do you remove almost everything from `.gitignore` file? Do you really need files like `.iml`?

Reply:
I guess, i've removed those lines by accident.